readme.md

```diff
@@ -110,6 +110,8 @@ If an element at the same key is present for both `x` and `y`, the value from
 
 Merging creates a new object, so that neither `x` or `y` is modified.
 
+**Note:** by default, arrays are concatenated rather than replacing the array values of `x` with `y`. If you want to change how this works then see `arrayMerge` section below.
```

Comment:
Probably better to replace the first sentence with the shorter "By default, arrays are merged by concatenating them."